projects/adt7420-pmdz/src/examples/dummy/dummy_example.c

```diff
@@ -62,13 +62,15 @@ int example_main()
 	float temp_now;
 	int ret;
 
+#ifndef LINUX_PLATFORM
```

Comment:
this commit title is `Added linux platform in project/adt7420-pmdz (untested)`, is this true ? this has to be tested